projects/fsastorefrontlib/src/cms-components/checkout/components/legal/legal-checkboxes/legal-checkboxes.component.html

```diff
@@ -20,7 +20,7 @@
     <span class="checkbox-text fs-icon icon-ok"></span>
   </label>
   <div class="row mt-5">
-    <div class="col-12 col-md-6 mb-3">
+    <div class="col-md-6 mb-3">
```

Comment:
No need for `col-12` class. On the smaller screens `div` takes full width, which corresponds to `col-1` class.